debezium-core/src/main/java/io/debezium/transforms/SchemaChangeEventFilter.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.transforms;
+
+import static org.apache.kafka.connect.transforms.util.Requirements.requireStruct;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.ConnectRecord;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.transforms.Transformation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.config.Configuration;
+import io.debezium.config.Field;
+import io.debezium.relational.history.ConnectTableChangeSerializer;
+import io.debezium.relational.history.HistoryRecord;
+import io.debezium.schema.SchemaChangeEvent;
+
+/**
+ * This SMT to filter schema change event
+ * @param <R>
+ */
+public class SchemaChangeEventFilter<R extends ConnectRecord<R>> implements Transformation<R> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SchemaChangeEventFilter.class);
+
+    private static final Field SCHEMA_CHANGE_EVENT_INCLUDE_LIST = Field.create("schema.change.event.include.list")
+            .withDisplayName("Schema change event include list")
+            .withType(ConfigDef.Type.LIST)
+            .withWidth(ConfigDef.Width.SHORT)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDescription(
+                    "Support filtering during DDL synchronization!");
+
+    private Set<SchemaChangeEvent.SchemaChangeEventType> includeSchemaChangeEvents;
+    private SmtManager<R> smtManager;
+
+    @Override
+    public void configure(Map<String, ?> configs) {
+        final Configuration config = Configuration.from(configs);
+        smtManager = new SmtManager<>(config);
+
+        final String includeSchemaChangeEvents = config.getString(SCHEMA_CHANGE_EVENT_INCLUDE_LIST);
+        if (includeSchemaChangeEvents == null) {
+            // throw exception
+            throw new IllegalArgumentException("The configuration [schema.change.event.include.list] cannot be empty");
+        }
+        this.includeSchemaChangeEvents = (includeSchemaChangeEvents == null)
```

Comment:
IIUC there lines above ensure that `includeSchemaChangeEvents` will be alway non-null.